src/decisionengine_modules/glideinwms/sources/factory_entries.py

```diff
@@ -40,10 +40,10 @@ def __init__(self, config):
         self.condor_config = config.get("condor_config")
         self.factories = config.get("factories", [])
         self._entry_gridtype_map = {
-            "Factory_Entries_Grid": ("gt2", "condor"),
-            "Factory_Entries_AWS": ("ec2",),
-            "Factory_Entries_GCE": ("gce",),
-            "Factory_Entries_LCF": ("batch slurm",),
+            "Grid": ["gt2", "condor"],
+            "AWS": ["ec2"],
+            "GCE": ["gce"],
+            "LCF": ["batch slurm"],
```

Comment:
This could become a module-level variable so that it can be imported in `publishers/fe_group_classads.py` and a list from its keys could be assigned to `GlideinWMSManifests.allow_types`.
Or a module with this constant could go in `glideinwms`.
The same could be imported in the test (or maybe is OK to redefine there)
All to avoid defining the same list in multiple places.

Reply:
If you prefer to leave it as it is (in the end the variable was defined in multiple places also before), I'm OK w/ it. Just add a comment or ping me on slack to let me know.

Reply:
Introduced module-level `ENTRY_TYPES` variable that can be used in `fe_group_classads`.